components/autocomplete/readme.md

```diff
@@ -62,6 +62,7 @@ If you want to provide a theme via context, the component key is `RTAutocomplete
 | `showSuggestionsWhenValueIsSet` | `Bool`                        | `false` | If true, the list of suggestions will not be filtered when a value is selected, until the query is modified. |
 | `suggestionMatch`               | `String`                      | `start` | Determines how suggestions are supplied. It can be `start` (query matches the start of a suggestion), `anywhere` (query matches anywhere inside the suggestion), `word` (query matches the start of a word in the suggestion) or `disabled` (disable filtering of provided source, all items are shown). |
 | `value`                         | `String`, `Array` or `Object` |         | Value or array of values currently selected component. |
+| `submitKeys`                    | `Array`                       | `[13]`  | Array of numbers representing keyboard keys that will trigger active item creation or selection. |
```

Comment:
Maybe it would be helpful to link to resource where possible values are listed? E.g. https://css-tricks.com/snippets/javascript/javascript-keycodes/

Reply:
Hi @3af, how about this one: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode?. Anyways, I have implemented the review changes...

Reply:
Hi @jesion, looks good too. Thanks for the changes.
Unfortunatelly I can't merge so let's wait till Javi has time for it.

Reply:
No probs @3af. Thanks for reviewing...